### ISSUE TITLE
Onboard packages for Sergio

### ISSUE THREAD

**Issue by lachmanfrantisek** [packit/packit-service#2398]
### What we are onboarding?

* [x] debconf - https://src.fedoraproject.org/rpms/debconf/pull-request/3
* [x]  debhelper - https://src.fedoraproject.org/rpms/debhelper/pull-request/1
* [x] debmirror - https://src.fedoraproject.org/rpms/debmirror/pull-request/2
* [x] dh-make - https://src.fedoraproject.org/rpms/dh-make/pull-request/1
* [x] dpkg - done in https://src.fedoraproject.org/rpms/dpkg/c/d53f6e218cb36d7e965d96b18e29f2af6c1cf838?branch=rawhide
* [x]  fakeroot - https://src.fedoraproject.org/rpms/fakeroot/pull-request/1
* [ ]  xpra - https://src.fedoraproject.org/rpms/xpra/pull-request/4
* [ ] virtualbox-guest-additions - waiting for pending release
* [ ] nextcloud - https://src.fedoraproject.org/rpms/nextcloud/pull-request/15, waiting for pending release
* [ ] clamav - waiting for pending release
* [x] smplayer - https://src.fedoraproject.org/rpms/smplayer/pull-request/2#
* [x] ImageMagick - https://src.fedoraproject.org/rpms/ImageMagick/pull-request/13

### Do you have more details?

Filled for @sergiomb2

**Comment by sergiomb2** [comment]
adding to list 
- clamav , which need remove form TARBALL unrar bits  `zcat ${TARBALL} | tar --delete -f - '*/libclamunrar/*' | xz -c > ${TARBALL_CLEAN} ` and other python script ( https://src.fedoraproject.org/rpms/clamav/blob/rawhide/f/update_clamav_data.sh )
 
- nextcloud

**Comment by lbarcziova** [comment]
PR for debconf package: https://src.fedoraproject.org/rpms/debconf/pull-request/3

**Comment by sergiomb2** [comment]
yah , I forgot that , virtualbox repo is svn even with one svn2git repo we don't have tags either ...

**Comment by sergiomb2** [comment]
I'd like add to List ImageMagick !

**Comment by sergiomb2** [comment]
Update:
ImageMagick we couldn't find a way to replace "-" by "." I forgot that in previous comment 

https://src.fedoraproject.org/fork/sergiomb/rpms/ImageMagick/blob/rawhide/f/.packit.yaml

https://release-monitoring.org/project/328484/ already replace "-" by "." and that may be a problem if so we can use 
https://release-monitoring.org/project/1372/

**Comment by lbarcziova** [comment]
Yes you are right, the ImageMagick is currently blocked on this issue: https://github.com/packit/packit/issues/2280.

So besides that, the missing packages are `opencv`, `clamav` and `git-extras`. If I check correctly, opencv and git-extras are up to date, so I would wait for the next release for them if you are not against. As for `clamav`, are you updating there only the `1.0.*` versions? I can see the latest proper version on Anitya is `1.3.1` ([here](https://release-monitoring.org/project/291/), and in rawhide `1.0.6`, so I am asking whether I should try to do that update?

**Comment by sergiomb2** [comment]
opencv have a new version https://bugzilla.redhat.com/show_bug.cgi?id=2290312 I don't  know if you want to try , opencv_clean.sh need to be run

**Comment by lbarcziova** [comment]
@sergiomb2 yes, I will try to onboard that today, thanks!

**Comment by sergiomb2** [comment]
may we add smplayer ? please

**Comment by lbarcziova** [comment]
@sergiomb2 I will have a look at it.
